lambda/index.py

```diff
@@ -108,6 +102,25 @@ def get_pg_usernames(cursor):
         rows.append(row[0])
     return rows
 
+def create_database(master_username, master_password, rds_host, rds_port, database_name):
+    conn = psycopg2.connect(user=master_username, password=master_password,
+                                host=rds_host, port=rds_port, dbname=database_name)
+    conn.autocommit = True
+    cursor = conn.cursor()
+
+    # Create database
+    create_database_flag = os.environ['CREATE_DATABASE']
+    if create_database_flag == "true":
```

Comment:
double logic exists here, you dont need to check the flag anymore since it was used to check before calling it